pkg/controller/clusterpool/clusterpool_controller_test.go

```diff
@@ -1107,7 +1107,9 @@ func TestReconcileClusterPool(t *testing.T) {
 			// DeletionTimestamp, isn't getting garbage collected by the fake client. This is the expected result once
 			// the issue is fixed:
 			// expectedTotalClusters: 2,
-			expectedTotalClusters:   3,
+			expectedTotalClusters: 3,
```

Comment:
Since you're adding the comment below, you can remove comment at line 1109

Reply:
Oh, they’re for different values. The comment below is pointing out that the bug referenced above doesn’t apply to the observed size.